src/components/index/GettingStarted.js

```diff
@@ -0,0 +1,87 @@
+/********************************************************************************
+ * Copyright (C) 2020 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import React from 'react'
+
+import styled from '@emotion/styled'
+import { breakpoints } from '../../utils/variables'
+import LearnMore from '../../resources/learn-more.svg'
+import TryIt from '../../resources/try-it.svg'
+import AdoptIt from '../../resources/download.svg'
+import Contribute from '../../resources/contribute.svg'
+
+import { Link } from 'gatsby'
+import Feature from './Feature'
+
+const StyledFeatures = styled.div`
+    .features {
+        padding: 1rem 0;
+    }
+
+    .feature__container {
+        display: flex;
+
+        @media(max-width: ${breakpoints.md}) {
+            flex-direction: column;
+        }
+    }
+`
+
+const features = [
+    {
+        img: <img src={TryIt} alt="Try it!" />,
+        title: "Try it!",
```

Comment:
minor: I would drop the `!` from each section title so it looks more professional, but I'll leave the decision up to you.